api-gateway/build.gradle.kts

```diff
@@ -12,4 +12,13 @@ dependencies {
     implementation(libs.spring.cloud.starter.kubernetes.client.config)
     implementation(libs.spring.security.core)
     implementation(projects.authenticationService)
+
+    // FixMe: we officially bring serialization CVE to our project
```

Comment:
Say hello to a back door!

Reply:
does the test failed on CI, without this?